apps/ca.c

```diff
@@ -82,7 +82,9 @@
 #define ENV_DATABASE            "database"
 
 /* Additional revocation information types */
+typedef int REVINFO_TYPE;
```

Comment:
Hi, 
My first version of this change was using an enum to restricted to a few particular values.
It's forbid by OpenSSL coding rules/style.
C standard doesn't define any particular implementation size for enum's, so some binary compatibility issues are possible.

In this case, as the enum will be restricted to one file and only with some static methods, it can't hurt anyone.